.circleci/config.yml

```diff
@@ -19,40 +19,42 @@ jobs:
           ACCEPT_EULA: 'yes'
           MSSQL_SA_PASSWORD: 5atyaNadella
```

Comment:
Maybe not related to this PR but we should change the hardcoded connection info to env vars.

Reply:
good point. for me it isn't so important bc we're spinning up a brand new SQL Server each time we run the test then destroying it after